pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -129,16 +161,29 @@ let
       ninja pkg-config
       python3WithPackages perl
       which
-      llvmPackages.bintools
+      buildPackages.${llvmPackages_attrName}.bintools
       bison gperf
     ];
 
+    depsBuildBuild = [
+      buildPlatformLlvmStdenv
+      buildPlatformLlvmStdenv.cc
+      pkgsBuildBuild.pkg-config
```

Comment:
pkgsBuildBuild. shouldn't be needed here because pkg-config has __spliced

Reply:
Indeed!  46ef180391e6